judges_seed.py

```diff
@@ -0,0 +1,238 @@
+    rows = table_contents.find_all("td")
+    return rows
+
+
+def get_bench_judges(url: str) -> list[str]:
+    """
+    Retrieves all bench judges
+    """
+    return [
+        item.get_text()
+        for index, item in enumerate(get_judge_rows(url))
+        if index % 2 == 1 and item.get_text() != "Name" and item.get_text()[0].isalpha()
+    ]
+
+
+def get_district_judges_magistrates(url: str) -> list[str]:
+    """
+    Retrieves all district judge magistrates
+    """
+    return [
+        item.get_text()
+        for index, item in enumerate(get_judge_rows(url))
+        if index % 3 == 0 and item.get_text() != "Judge"
+    ]
+
+
+def get_diversity_high_court_judges(url: str) -> list[str]:
+    """
+    Retrieves diversity and community judges or high court judges
+    """
+    return [
+        item.get_text()
+        for index, item in enumerate(get_judge_rows(url))
+        if index % 2 == 0 and item.get_text() != "Name" and item.get_text()[0].isalpha()
+    ]
+
+
+def get_judge_advocates(url: str) -> list[str]:
+    """
+    Retrieves all judge advocates
+    """
+    return [
+        item.get_text()
+        for index, item in enumerate(get_judge_rows(url))
+        if index % 2 == 0 and item.get_text() != "Judge"
+    ]
+
+
+def get_circuit_district_judges(url: str) -> list[str]:
+    """
+    Retrieves all circuit district judges
+    """
+    rows = get_judge_rows(url)
+    names = []
+    for index, item in enumerate(rows):
+        if index % 3 == 0 and item.get_text() != "Judge":
+            name = item.get_text()
+            if not item.get_text()[-1].isalpha():
+                name = name[:-1]
+            names.append(name)
+    return names
+
+
+def gather_all_judges() -> list[str]:
+    """
+    Retrieves and combines all judges names
+    """
+    high_court_masters = get_diversity_high_court_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/hc-masters-list/"
+    )
+
+    bench_chairs = get_bench_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/bench-chairmen-list/"
+    )
+
+    district_judges_magistrates = get_district_judges_magistrates(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/dj-mags-ct-list/"
+    )
+
+    diversity_community_judges = get_diversity_high_court_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/diversity-and-community-relations-judges-list/"
+    )
+
+    judge_advocates_general = get_judge_advocates(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/jag-list/"
+    )
+
+    circuit_judges = get_circuit_district_judges(
+        "https://www.judiciary.uk/about-the-judiciary/who-are-the-judiciary/list-of-members-of-the-judiciary/circuit-judge-list/"
+    )
+
```

Comment:
Lots of DRY here, maybe could have saved base url and changed part of path that has variation?